src/uniprotkb/components/landing-page/StatisticsChart.tsx

```diff
@@ -1,230 +1,101 @@
-    .style('fill', 'none')
-    .style('opacity', 0.3)
-    .style('stroke', 'black')
-    .style('stroke-width', '2px')
-    .transition()
-    .duration(1000)
-    .attrTween('points', (d) => {
-      const interpolate = d3.interpolate(current, d);
-      current = interpolate(0);
-      return (t) => {
-        const d2 = interpolate(t);
-        const pos = outerArc.centroid(d2);
-        pos[0] = radius * 0.95 * (midAngle(d2) < Math.PI ? 1 : -1);
-        const pointsArray = [midArc.centroid(d2), outerArc.centroid(d2), pos];
-        return pointsArray.map((point) => point.join(',')).join(' ');
-      };
-    });
+import PieChart, { StatisticsGraphItem } from '../graphs/PieChart';
 
-  polyline.exit().remove();
+type StatisticsChartProps = {
+  releaseNumber?: string;
+  reviewed: boolean;
+  unreviewed: boolean;
 };
 
-const StatisticsChart = ({ releaseNumber }: { releaseNumber?: string }) => {
-  const history: History = useHistory();
-  const svgRef = useRef<SVGSVGElement>(null);
-
+const StatisticsChart = ({
+  releaseNumber,
+  reviewed,
+  unreviewed,
+}: StatisticsChartProps) => {
   const reviewedStats = useDataApi<StatisticsPayload>(
     releaseNumber &&
-      `${API_PREFIX}/statistics/releases/${releaseNumber}/reviewed`
+      stringifyUrl(
+        `${API_PREFIX}/statistics/releases/${releaseNumber}/reviewed`,
+        { categories: 'superkingdom' }
+      )
   );
 
   const unreviewedStats = useDataApi<StatisticsPayload>(
     releaseNumber &&
-      `${API_PREFIX}/statistics/releases/${releaseNumber}/unreviewed`
+      stringifyUrl(
+        `${API_PREFIX}/statistics/releases/${releaseNumber}/unreviewed`,
+        { categories: 'superkingdom' }
+      )
   );
 
-  useEffect(() => {
-    if (
-      svgRef.current &&
-      reviewedStats.data?.results &&
-      unreviewedStats?.data?.results
-    ) {
-      const taxonReviewed = reviewedStats.data.results.find(
-        (category) => category.categoryName === 'SUPERKINGDOM'
-      );
-      const taxonUnreviewed = unreviewedStats.data.results.find(
-        (category) => category.categoryName === 'SUPERKINGDOM'
-      );
-      const taxonSummed = Object.values(
-        [
-          ...(taxonReviewed?.items as []),
-          ...(taxonUnreviewed?.items as []),
-        ].reduce(
-          (
-            acc: Record<string, StatisticsItem>,
-            { name, count, entryCount }
-          ) => {
-            acc[name] = {
-              name,
-              count: (acc[name] ? acc[name].count : 0) + count,
-              entryCount: (acc[name] ? acc[name].entryCount : 0) + entryCount,
-            };
-            return acc;
+  const data: StatisticsGraphItem[] | undefined = useMemo(() => {
+    if (reviewedStats.loading || unreviewedStats.loading) {
+      return undefined;
+    }
+    const taxonSummed = Object.fromEntries(
+      Array.from(nameToQueryKingdoms.keys(), (name) => [
+        name,
+        {
+          name,
+          entryCount: 0,
+          to: {
```

Comment:
Good catch, there was a typo in the Location object I was generating. Somehow I'm surprised that TS wasn't complaining about it. It should be fixed now

Reply:
Do you think each slice of the pie should point to the group by page for that taxon?
I was thinking that maybe with the title of the section pointing to the common ancestor of all taxons in the graph that would be good enough.
Here we don't have any number associated with the pie slices, so we can make these links point to anywhere, even simply to the taxonomy page for it. Although I was also thinking that this being the UniProtKB landing page it would be important that most links point to searches within that namespace, and we coud have links to specific taxons whenever we work on a landing page for the taxonomy namespace.